main.go

```diff
@@ -5,15 +5,13 @@ import (
 	"fmt"
 	"net/http"
 	"os"
-	"pebble-dev/rebblestore-api/db"
 
 	"github.com/gorilla/handlers"
 	_ "github.com/mattn/go-sqlite3"
 	"github.com/pborman/getopt"
-	//rsapi "github.com/pebble-dev/rebblestore-api"
-
-	"pebble-dev/rebblestore-api/common"
-	"pebble-dev/rebblestore-api/rebbleHandlers"
+	"github.com/pebble-dev/rebblestore-api/common"
+	"github.com/pebble-dev/rebblestore-api/db"
+	"github.com/pebble-dev/rebblestore-api/rebbleHandlers"
```

Comment:
Why link to the repo instead of using the local files? This doesn't work when a package is modified locally.

Same goes for the other files where this has happened.

Specifically, I cannot even run your branch because of this:

```
../../github.com/pebble-dev/rebblestore-api/rebbleHandlers/application.go:251: not enough arguments in call to ctx.Database.GetAllApps
        have ()
        want (string, bool, int, int)
```

Reply:
Ok, I've changed it all back, I didn't understand the structure that we had, but I do now and it makes sense. Thanks for catching it!